jscomp/others/js_date.ml

```diff
@@ -123,7 +123,7 @@ external toGMTString : t -> string = "" [@@bs.send]
 [@@ocaml.deprecated "use `toUTCString` instead"]
 
 external toISOString : t -> string = "" [@@bs.send]
-external toJSON : t -> string = "" [@@bs.send]
+external toJSON : t -> string option = "" [@@bs.send] [@@bs.return {null_to_opt}]
```

Comment:
good news is `string option` is good enough, in the master, it is essentially `string | undefined` so `[@@bs.return null_to_opt]` is not needed any more.

Reply:
Ah, ignore my previous comment, here it returns `null`